yaml/parse_test.ts

```diff
@@ -333,9 +333,9 @@ Deno.test({
       -Infinity,
       NaN,
       12000,
-      75,
-      4520,
-      -4520,
+      1,
+      1,
+      -1,
```

Comment:
These need to be:
```
"1:15",
"1:15:20",
"-1:15:20",
```

(all need to be resolved as strings) if we drop the sexagesimal integer parsing.